week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,7 +22,15 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(currencySymbol,price) {
```

Comment:
The all tests for this function are failing which means your code has problems. Please fix.